MekHQ/src/mekhq/campaign/parts/FireControlSystem.java

```diff
@@ -72,15 +72,53 @@ public void updateConditionFromEntity(boolean checkForDestruction) {
 			hits = ((Aero)unit.getEntity()).getFCSHits();
 			if(checkForDestruction 
 					&& hits > priorHits 
-					&& Compute.d6(2) < campaign.getCampaignOptions().getDestroyPartTarget()) {
+					&& (hits < 3 && !campaign.getCampaignOptions().useAeroSystemHits())
+                    && Compute.d6(2) < campaign.getCampaignOptions().getDestroyPartTarget()) {
 				remove(false);
 				return;
-			}
+			} else if (hits >= 3) {
+                remove(false);
+                return;
+            }
 		}
 	}
 	
 	@Override 
 	public int getBaseTime() {
+        if (campaign.getCampaignOptions().useAeroSystemHits()) {
+            //Test of proposed errata for repair times
+            Entity e = unit.getEntity();
+            if (isSalvaging()) {
+                if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+                    if (e.hasNavalC3()) {
+                        return 8640;
+                    }
+                    return 4320;
+                } else {
+                    return 1080;
+                }
+            }
+            if (hits == 1) {
+                if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+                    if (e.hasNavalC3()) {
+                        return 480;
+                    }
+                    return 240;
+                } else {
+                    return 120;
+                }
+            } 
+            if (hits == 2) {
+                if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+                    if (e.hasNavalC3()) {
+                        return 960;
+                    }
```

Comment:
I also see a lot of inconsistency in repair times. Here, the base repair time for AeroSpace is 240, the larger time for DS/JS/WS is 480 (double), and the Naval C3 doubles that again. Yet in other places, the DS/JS/WS value is **much** more than double... it would be nice to have a consistent formula, and then just apply said formula at the end of the method after setting the base value. This is feedback on the proposed errata to have a formula like that, rather than pulling values out of thin air.

Reply:
There is a rhyme and reason for my times. I left the large craft times as published in SO. I happen to agree with you heartily on their being absurd in some cases, but it was my understanding that RAW is RAW and those get left alone. 
December 2017 errata added the small craft section to aero repairs, which includes sensors and life support but not FCS or avionics, so I put those into this. 
SO as published has repair times for all of these systems, which typically don't get used in MM/MHQ because the system gets destroyed and requires replacement on a roll of 2-9 by default. I took these times into account and tried to make them realistic increments for 1 hit/2hits/3hits where the system is not destroyed. In a lot of cases they're really small, so I referenced Mech repair times in order to make the numbers for small craft less than those for large craft. 
I was thinking that repair time should go up when a system takes multiple hits, but maybe a different option would be for the difficulty modifier to go up. Isn't that how ground unit repairs work, for things like IS and Engines?
If we want to change this to be an unofficial 'sanity check' on large craft repair times, I'd be more than happy to reduce some of the absurd numbers like 6720 and 4320.  I think 5-10x the small craft value is fair. WWII repair times DID happen with 24 hour work in shifts, lots of extra people and a shipyard. SO covers all those things, once the planets project identifies systems with shipyards/factories/stations with repair facilities and I get docking implemented.

Reply:
Thats fair, but even a warships damage control parties are more than one man, and more than one party for that matter. Hence my comments that only getting the equivalent of 1 crew member worth of time is... dumb... and that the time should be based on number of crew. Would also give an advantage to carrying more crewmen than the minimum required